TREnvironmentEditor/EMEditorMapping.cs

```diff
@@ -93,6 +95,10 @@ public void AlternateTextures()
             {
                 OneOf.ForEach(s => s.RemapTextures(AlternativeTextures));
             }
+            if (ConditionalOneOf != null)
+            {
+                ConditionalOneOf.ForEach(s => s.RemapTextures(AlternativeTextures));
```

Comment:
You can do `ConditionalOneOf?.ForEach(s => s.RemapTextures(AlternativeTextures));` and remove the `if != null` here, provided you have the right language version. Can do the same for the others here too.